build/csv/csv-gen.sh

```diff
@@ -37,7 +37,20 @@ function generate_csv() {
 
     "${YQ_CMD_MERGE[@]}" "$CSV_FILE_NAME" "$ASSEMBLE_FILE_OCP"
 
+    # We don't need to include these files in csv as ocs-operator creates its own.
+    rm -rf "../../build/csv/ceph/$PLATFORM/manifests/rook-ceph-operator-config_v1_configmap.yaml"
+
+    # This change are just to make the CSV file as it was earlier and as ocs-operator reads.
+    # Skipping this change for darwin since `sed -i` doesn't work with darwin properly.
```

Comment:
```suggestion
    # Skipping this change for darwin since `sed -i` doesn't work with darwin properly.
    # and the csv is not ever needed in the mac builds.
```